### PR TITLE
 hard coded security questions && answers for test users || backup added #3846

### PR DESCRIPTION
**Pull request by KaliGreen**
Security Questions and answers were hard coded for test users, so that we can be able to run fixes when user can't login

### CODE REVIEW

**Review by Zangetsu101 — APPROVED** [review]
The changes look good but we might want use answers that are a bit more meaningful

**Comment by rikukissa** [comment]
Agreed. We need to consider, that someone might be demoing the password recovery route. How about something like

Birth town: Lusaka
High school: Lusaka High School
Favourite movie: Star wars

**Comment by Zangetsu101** [comment]
Sounds good. @KaliGreen Could you please make the suggested changes to the answers please?

**Comment by KaliGreen** [comment]
Alright will attend to it

**Review by Zangetsu101 — APPROVED** [review]

**Comment by euanmillar** [comment]
@rikukissa @naftis @Zangetsu101 in the end I decided to set the security question answers to be the same as the login password.  If a country is configuring this then they do not want to use "Lusaka High School" or "Lusaka" for their test user security answers as those are Farajaland specific.  So I took the opportunity to address https://github.com/opencrvs/opencrvs-core/issues/3680 "Make all command like tools configurable via named parameters" ... You now run the populate script like this:  `yarn db:populate --password_for_users=test --environment=development --alpha3_country_code=ZMB` I improved the error messaging to validate the inputs and ensure that it is clear that the value for "--password_for_users" is also applied to the security question answers.

**Review by euanmillar — APPROVED** [review]

File: db-populate.sh [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+#
+# OpenCRVS is also distributed under the terms of the Civil Registration
+# & Healthcare Disclaimer located at http://opencrvs.org/license.
+#
+# Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+# graphic logo are (registered/a) trademark(s) of Plan International.
+
+print_usage_and_exit () {
+    echo 'Usage: ./db-populate.sh --password_for_users=XXX --environment=development|production --alpha3_country_code=XXX'
+    echo "  --password_for_users will be the password and security question answer for all users used in development."
```
Comment (euanmillar) [inline db-populate.sh:14]: This comment shows how the password will be used

**Pull request merged**